src/content/docs/style-guide/word-choice/pricing-language-guidelines.mdx

```diff
@@ -44,10 +44,9 @@ Some notes on style and formatting for pricing-related language:
   * When referencing a model, it's good practice to point to either the doc for that model, or else the [doc explaining the differences between the models](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-changes-pricing-user-model/#pricing-user-table), whichever makes more sense.
 * "usage-based." Our newer pricing model can be referred to as “usage-based” or “consumption-based,” but “usage-based” is the most commonly used and more preferred term.
 * Editions:
-  * Our pricing editions are not "plans" or "models" or "tiers"; they are "editions".
-  * We don't refer to our [free tier](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing#free) as an edition. We just call it "free tier."
-  * Our pricing editions are formatted with title case: Standard, Pro, and Enterprise. Example use: _If your organization is on the [Pro edition](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions)..._
-  * A New Relic organization can only have a single edition.
+  * Our [pricing editions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) are not "plans" or "models" or "tiers"; they are "editions". One exception is that it's acceptable to refer to the Free edition as the "free tier" simply because we have historically used that language a lot. 
```

Comment:
```suggestion
  * Our [pricing editions](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#editions) are not "plans" or "models" or "tiers"; they are "editions". One exception is that it's acceptable to refer to the Free edition as the "free tier" simply because we have used that label in the past. 
```